src/Tween.js

```diff
@@ -429,6 +429,7 @@ export default class Tween extends PIXI.utils.EventEmitter {
      * @fires PIXI.tween.Tween#repeat
      *
      * @param {number} deltaMS - Time elapsed in milliseconds from last update to this update.
+     * @param {number} progress - 0-1 decimal value representing proportion of completion.
```

Comment:
this is docs for the update function param, not for the event. What you need to update docs wise is near the top of the file